es/13_1_Escribiendo_Puzzle_Scripts.md

```diff
@@ -0,0 +1,523 @@
+# 13.1: Escribir guiones de rompecabezas
+
+Los scripts de Bitcoin realmente _no_ tienen que depender del conocimiento de una clave secreta. En cambio, pueden ser rompecabezas de cualquier tipo.
+
+## Escribir guiones de álgebra simples
```

Comment:
`## Escribir scripts de álgebra simples`. I think that `guion` introduces some ambiguity.